www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -686,6 +686,14 @@
         <default>0</default>
         <Required>Y</Required>
       </https_only>
+      <ocsp_stapling type="BooleanField">
+        <default>0</default>
+        <Required>Y</Required>
```

Comment:
this would need a model version bump at the top of this file...

Reply:
I think it is better if I do that after merge

Reply:
ok, fine by me